front/src/modules/trainschedule/components/ManageTrainSchedule/ManageTrainScheduleMap/AddPathStepPopup.tsx

```diff
@@ -73,33 +67,25 @@ function RenderPopup({ pathProperties }: RenderPopupProps) {
       setTrackOffset(offset);
     };
 
-    if (featureInfoClick.displayPopup) {
+    if (featureInfoClick) {
       calculateOffset();
     }
   }, [featureInfoClick]);
 
-  if (
```

Comment:
You can keep the check for `featureInfoClick.feature.properties` I think and remove the `?` below

Reply:
Done